Tools/mark-shipped.ps1

```diff
@@ -4,16 +4,18 @@ param ()
 Set-StrictMode -version 2.0
 $ErrorActionPreference = "Stop"
 
-function MarkShipped([string]$dir) {
-    $shippedFilePath = Join-Path $dir "PublicAPI.Shipped.txt"
+function MarkShipped([string]$dir, [string]$access) {
```

Comment:
would we want to have default values?

Reply:
or fail if one is missing, maybe.

Reply:
The parameters are now mandatory and function calls will fail without both